packages/redux_persist_flutter/lib/redux_persist_flutter.dart

```diff
@@ -95,9 +112,62 @@ class SharedPreferencesEngine implements StorageEngine {
   @override
   Future<void> save(Uint8List? data) async {
     final sharedPreferences = await _getSharedPreferences();
-    sharedPreferences.setString(key, uint8ListToString(data)!);
+    sharedPreferences.setString(key, await uint8ListToString(data) ?? '');
```

Comment:
Keep it consistent with what they've already got. Do this instead:
`(await uint8ListToString(data))!`